### PR TITLE
feat: Local iterations

### PR DESCRIPTION
**Pull request by dnys1**
Initial release of the Celest CLI with a full local iterations experience.

### CODE REVIEW

**Comment by dnys1** [comment]
Current dependencies on/for this PR:
* `main`
  * **PR #5** <a href="https://app.graphite.dev/github/pr/celest-dev/celest/5?utm_source=stack-comment-icon" target="_blank">Graphite</a>  👈

This [stack of pull requests](https://stacking.dev/?utm_source=stack-comment) is managed by [Graphite](https://app.graphite.dev/github/pr/celest-dev/celest/5?utm_source=stack-comment).

File: packages/celest/example/README.md [hunk 1]
```diff
@@ -0,0 +1,8 @@
+# hello
+
+A new Flutter project.
+
+## Running the app
+
+- Go to the hello directory, and run `flutter pub get`
+- Go to the `celest` directory inside the hello example app and run `dart pub get`
```
Comment (abdallahshaban557) [inline packages/celest/example/README.md:8]: just wondering here on why you need to run dart pub get if the CLI takes care of installing dependencies for you?

File: packages/celest/example/README.md [hunk 2]
```diff
@@ -0,0 +1,8 @@
+# hello
+
+A new Flutter project.
```
Comment (abdallahshaban557) [inline packages/celest/example/README.md:3]: This is a new Flutter project.

File: packages/celest/example/README.md [hunk 3]
```diff
@@ -0,0 +1,8 @@
+# hello
```
Comment (abdallahshaban557) [inline packages/celest/example/README.md:1]: Flutter project

File: packages/celest/example/README.md [hunk 4]
```diff
@@ -0,0 +1,8 @@
...
+
+A new Flutter project.
+
+## Running the app
+
+- Go to the hello directory, and run `flutter pub get`
```
Comment (abdallahshaban557) [inline packages/celest/example/README.md:7]: Go to the `hello` directory in your console, and run `flutter pub get`.

File: packages/celest/example/celest/project.dart [hunk 5]
```diff
@@ -0,0 +1,5 @@
+import 'package:celest/celest.dart';
+
+const project = Project(
+  name: 'hello',
```
Comment (abdallahshaban557) [inline packages/celest/example/celest/project.dart:4]: I feel like the name of the project and the app should be something like "my_flutter_project". Hello is confusing me. That should change both on the frontend and backend.

**Review by abdallahshaban557 — APPROVED** [review]
LGTM

**Pull request merged**